Intersect.Client/Entities/Player.cs

```diff
@@ -805,14 +805,23 @@ private void RetreiveBagItemInputBoxOkay(object sender, EventArgs e)
         //Trade
         public void TryTradeItem(int index)
         {
-            if (ItemBase.Get(Inventory[index].ItemId) != null)
-            {
-                if (Inventory[index].Quantity > 1)
-                {
-                    var iBox = new InputBox(
-                        Strings.Trading.offeritem,
-                        Strings.Trading.offeritemprompt.ToString(ItemBase.Get(Inventory[index].ItemId).Name), true,
-                        InputBox.InputType.NumericSliderInput, TradeItemInputBoxOkay, null, index, Inventory[index].Quantity
+            var slot = Inventory[index];
+            var quantity = slot.Quantity;
+            var tradingItem = ItemBase.Get(slot.ItemId);
+            if (tradingItem != null)
+            {
+                if (quantity > 1)
```

Comment:
And tbh to further reduce nesting I kind of want to invert this check (if quantity == 1), do the packet send and return to keep InputBox out of an if.

Reply:
👍🏽